amundsen_application/static/js/components/PreferencesPage/PreferenceGroup/index.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+export type PreferenceGroupProps = {
+  changePreference: any;
```

Comment:
nit: Prefer to call this `onChange` or `onClick`. There are places where I know we aren't particularly following this, but if we know a method is executed on a "click", "submit", "hover", etc new folks will have an easier time if we start to name these methods base on how it will be used instead of tying the name to an understanding of the component.

not a nit: Let's type this more appropriately as `(value: string) => any`. Right now this allows us to not even pass a method, and we could set `changePreference: 1000` and break the component.